.github/CODEOWNERS

```diff
@@ -1,14 +1,13 @@
 # chain-interactions
 /crates/sncast/             @foundry-rs/starknet-foundry-cast
+/sncast_std                 @foundry-rs/starknet-foundry-cast
 
-# snforge-cli
-/crates/forge               @foundry-rs/starknet-foundry-forge-cli
-/crates/forge-runner        @foundry-rs/starknet-foundry-forge-cli
-
-# snforge-core
-/crates/runtime             @foundry-rs/starknet-foundry-forge-core
-/crates/cheatnet            @foundry-rs/starknet-foundry-forge-core
-/crates/forge/src/running.rs @foundry-rs/starknet-foundry-forge-core
-/crates/forge/src/cheatcodes_hint_processor.rs @foundry-rs/starknet-foundry-forge-core
-/crates/forge/tests/integration @foundry-rs/starknet-foundry-forge-core
-/crates/sierra-casm         @foundry-rs/starknet-foundry-forge-core
+# snforge
+/crates/forge               @foundry-rs/starknet-foundry-forge
+/crates/forge-runner        @foundry-rs/starknet-foundry-forge
+/crates/runtime             @foundry-rs/starknet-foundry-forge
```

Comment:
can we add cast team to runtime and conversions as well? We use/will use them in our codebase, and it would be cool to know what's hapening there 🙏

Reply:
Done